### PR TITLE
[Australia] - Publish conference recap and links

### PR DESCRIPTION
**Pull request by swapnilogale**
----
📚 Documentation preview 📚: https://writethedocs-www--2071.org.readthedocs.build/

### CODE REVIEW

**Comment by swapnilogale** [comment]
Waiting on videos to be uploaded and processed ⏳

File: docs/conf/australia/2023/news/thanks-recap.rst [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+None of our organizers were approached about any CoC concerns at this event.
+We know that this is, in part, a reflection of the thoughtful and inclusive community that we're so lucky to have.
+
+That being said, we're also very conscious of the fact that just because nothing was reported doesn't mean nothing happened.
+If there was any personal conduct or other aspect of the conference experience that did not meet the expectations of behaviour set out in the CoC, please email us at `conduct@writethedocs.org <mailto:conduct@writethedocs.org>`_.
+
+Ready for more?
+===============
+
+We are already in 2024 folks! Our `Portland conference <https://www.writethedocs.org/conf/portland/2024/>`__ is scheduled to happen 14-16 May and CFPs are now open.
```
Comment (plaindocs) [inline docs/conf/australia/2023/news/thanks-recap.rst]: I'd say "the CFP" or "the call for proposals"

File: docs/conf/australia/2023/news/thanks-recap.rst [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+You can `submit a talk for the Portland conference <https://www.writethedocs.org/conf/portland/2024/cfp/>`__ until **midnight Pacific time 15 January 2024**.
+This conference will take place in-person in Portland.
+
+If you cannot make the Portland conference, we will also announce dates for the online Atlantic conference soon.
+
+Around mid 2024, we will announce dates and more details for the Australian conference.
```
Comment (plaindocs) [inline docs/conf/australia/2023/news/thanks-recap.rst]:
```suggestion
We will announce dates and more details for the next Australian conference in the middle of 2024.
```

File: docs/conf/australia/2023/news/thanks-recap.rst [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+meetups in cities across the country, and if you don’t see your city listed consider `starting`_ a meetup!
+
+.. _build the meetup community: http://www.writethedocs.org/meetups/
+.. _starting: http://www.writethedocs.org/organizer-guide/meetups/starting/
+
+Thanks again
+============
+
+We're so grateful to our sponsors, speakers, volunteers, and attendees for making this conference possible.
+Whether you were able to come out this time or not, we hope to see you again next year – or even sooner at one of our other events!
```
Comment (plaindocs) [inline docs/conf/australia/2023/news/thanks-recap.rst:121]: Were there sponsors? We should list them in this post?

**Review by plaindocs — CHANGES_REQUESTED** [review]
Few minor nits

**Comment by plaindocs** [comment]
@swapnilogale I was thinking more like this https://github.com/writethedocs/www/pull/2068/files#diff-270d41847245bf5dc3d34d4f475dca1bbf64bb319f5a33f7ac90b4fdb1c59b06R71-R84 but up to you

**Review by plaindocs — APPROVED** [review]

**Comment by swapnilogale** [comment]
> @swapnilogale I was thinking more like this https://github.com/writethedocs/www/pull/2068/files#diff-270d41847245bf5dc3d34d4f475dca1bbf64bb319f5a33f7ac90b4fdb1c59b06R71-R84 but up to you

Yep that works too. I'll make the change later today and publish. Videos are all scheduled to go live at 5pm today AEDT, so I'll fix it before then and push.

**Pull request merged**